src/utils/openmc/universe.py

```diff
@@ -911,8 +880,24 @@ def create_xml_subelement(self, xml_element):
         if not test is None:
             return
 
-        lattice_subelement = \
```

Comment:
It's not a big deal to me, I understand your reasoning, so you can leave it out of this PR if you wish.